README.md

```diff
@@ -96,17 +96,20 @@
   >}
   >```
   >On initialization, the plugin well call the function and store the returned Media tag to request the VAST/VPAID ads.
-  
+
 ### playAdAlways
   >Flag to indicate if we must play an ad whenever possible. If set to true the plugin will play an ad every time the user watches a new video or replays the actual video.
   >Defaults to false
-  
+
 ## adCancelTimeout
  >Number of milliseconds for the ad to start before canceling it. Defaults to 3000
- 
+
 ## adsEnabled
  >Flag to disable the ads. Defaults to false.
- 
+
+## autoResize
+ >Flag to enable resize when the event `resize` or `orientationchange` is trigger by the the browser. Defaults to true
+
```

Comment:
I think the description is not very clear.

Flag to enable resize of the adUnit on window's `resize` and `orientationchange` events. This is useful for responsive players. Defaults to true